csrc/mlp_cuda.cu

```diff
@@ -1274,28 +1271,29 @@ int mlp_fp(
 
     // try with cublaslt first for supported case with valid handle
     int cublaslt_status = 1;
-    if(lthandle_status == CUBLAS_STATUS_SUCCESS && activation < 2){
-      cublaslt_status = mlp_gemm_lt(
-        ltHandle,
-        CUBLAS_OP_T,
-        CUBLAS_OP_N,
-        ofeat,
-        batch_size,
-        ifeat,
-        &one,
-        weight,
-        ifeat,
-        input,
-        ifeat,
-        &zero,
-        output,
-        ofeat,
-        lt_workspace,
-        1 << 22,
-        stream,
-        use_bias == 1,
-        activation == 1,
-        bias);
+    if(activation < 1){
+        cublaslt_status = mlp_gemm_lt(
+          //ltHandle,
+          (cublasLtHandle_t)handle,
+          CUBLAS_OP_T,
+          CUBLAS_OP_N,
+          ofeat,
+          batch_size,
+          ifeat,
+          &one,
+          weight,
+          ifeat,
+          input,
+          ifeat,
+          &zero,
+          output,
+          ofeat,
+          lt_workspace,
+          1 << 22,
```

Comment:
what's with the magic number here?  Is it the same magic number as is at https://github.com/seryilmaz/apex/blob/seryilmaz/mlp_fixes2/csrc/mlp.cpp#L67 ?  Can those please be combined and given a name if so?

Reply:
Yeah they are the same number. The idea at the time is to use something large enough for small-ish gemm, since those are the sizes mlp extension targeting to help, while also not too big so we don't care enough to try shrink and save.
agree with given it a name